### ISSUE TITLE
Getting HTTP/404 for WebAPI/user/* routes

### ISSUE THREAD

**Issue by tstringer-fn** [OHDSI/WebAPI#2292]
### Expected behavior

Successful authentication.

### Actual behavior

Failed authentication. It appears as though Atlas is getting HTTP/404 from WebAPI for the following routes:

* `WebAPI/user/me`
* `WebAPI/user/login/db`

I can verify this by running a local `curl` against WebAPI (away from Atlas) and noticing that WebAPI is in fact giving HTTP/404 for these routes.

I have setup security with the [security configuration documentation](https://github.com/OHDSI/WebAPI/wiki/Security-Configuration) but ended up in this state and unsure how to troubleshoot further.

### Steps to reproduce behavior

I'm am unsure how to repro this, as I follow the documented for security setup.

Any help is much appreciated!

**Comment by konstjar** [comment]
First question, can you see response from WebAPI/info endpoint? Just to validate that WebAPI is up and running.

**Comment by tstringer-fn** [comment]
Yes! Sorry, I should've mentioned that. I do see a successful response from `WebAPI/info`.

**Comment by eddyfrankenberger** [comment]
Hi @konstjar, any thoughts as to what may be occurring here? I'm seeing the same behavior.

**Comment by konstjar** [comment]
HTTP/404 response to `WebAPI/user/login/db` could come in one case when `security_provider` property is not set correctly. Could you please check that you have `AtlasRegularSecurity` value set?

**Comment by eddyfrankenberger** [comment]
The `AtlasRegularSecurity` value set is set, and I see the 404 response in Chrome when navigating to these endpoints as well.

**Comment by konstjar** [comment]
In this case you should get 401 for `/WebAPI/user/me` endpoint in Chrome. Only that endpoint has `GET` request type.

**Comment by eddyfrankenberger** [comment]
Hi @konstjar please see attached image of 404 errors in Chrome when navigating to `/WebAPI/user/me`
![image (6)](https://github.com/OHDSI/WebAPI/assets/115470491/35023f87-0ff6-445f-abe3-ac0155c548e3)

**Comment by konstjar** [comment]
Just to avoid confusion, did you re-build the WebAPI after pom.xml changes? It looks like application does not see that `security.provider` property as `AtlasRegularSecurity`.

**Comment by eddyfrankenberger** [comment]
I did re-try building the WAR once more after confirming the `security.provider` property was set to `AtlasRegularSecurity` in the settings XML file, but I'm still seeing the same behavior.
